pkg/secret/secret.go

```diff
@@ -90,6 +106,26 @@ func Pull(apiClient *clients.Settings, name, nsname string) (*Builder, error) {
 	return &builder, nil
 }
 
+// Get fetches the defined secret from the cluster.
+func (builder *Builder) Get() (*corev1.Secret, error) {
```

Comment:
Please remove this function. It's not needed for standard k8s objects. For standard k8s objects we have full client interface support and we use Get method directly in Exist.